sample_apps/rust/examples/query_sample.rs

```diff
@@ -0,0 +1,220 @@
+use aws_sdk_timestreamquery as timestream_query;
+use clap::Parser;
+use std::error::Error;
+use std::fs;
+pub mod utils;
+use crate::utils::query_common;
+
+async fn execute_sample_queries(
+    region: String,
+    database_name: String,
+    table_name: String,
+    f: &fs::File,
+) -> Result<(), timestream_query::Error> {
+    let client = query_common::get_connection(&region).await.unwrap();
```

Comment:
In valkey-glide, we have a strict no `unwrap` policy.  I don't know what policy your team uses, but you can always use `expect` instead.

Reply:
Changed to `unwrap`. What is the reason for the strict no `unwrap` policy? Does unwrapping assume the result is `Ok`?